web/annotation_layer_builder.css

```diff
@@ -25,6 +29,12 @@
   .annotationLayer .buttonWidgetAnnotation.radioButton input:required {
     outline: 1.5px solid selectedItem;
   }
+  :root {
+    --inner-focus-line-color: CanvasText;
+    --input-unfocus-border-color: ActiveText;
+    --input-disabled-border-color: GrayText;
+    --input-hover-border-color: Highlight;
+  }
```

Comment:
I'd suggest moving this `:root` declaration to be placed *first* within the `@media (forced-colors: active) {`-block.